CHANGELOG.md

```diff
@@ -73,6 +73,7 @@ The following new configuration options are available.
 - [#8394](https://github.com/influxdata/influxdb/pull/8394): Optimize top() and bottom() using an incremental aggregator.
 - [#7129](https://github.com/influxdata/influxdb/issues/7129): Maintain the tags of points selected by top() or bottom() when writing the results.
 - [#8188](https://github.com/influxdata/influxdb/issues/8188): Write CQ stats to _internal
+- [#8495](https://github.com/influxdata/influxdb/pull/8495): Improve CLI connection warnings
```

Comment:
Can you move this up to the 1.5 section?

Reply:
Done! Thanks.